controllers/scopeinstance_controller.go

```diff
@@ -115,6 +146,16 @@ func (r *ScopeInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	requirement, err := labels.NewRequirement(scopeInstanceHashKey, selection.NotEquals, []string{util.HashObject(in.Spec)})
 	if err != nil {
+		cErr := r.updateScopeInstanceCondition(ctx, in, metav1.Condition{
+			Type:               operatorsv1.ScopeInstanceSucceededType,
+			Status:             metav1.ConditionFalse,
+			ObservedGeneration: in.Generation,
+			Reason:             "NewRequirementLabelFailure",
```

Comment:
I didn't make this one a constant yet as I am not sure this error case is one that users will generally need to know about, but wanted to avoid removing it until there was more feedback on it. Thoughts?

Reply:
Resolved in commit https://github.com/awgreene/oria-operator/pull/21/commits/01d4b49156e5523bab706fd1c4c06ee6d1fa057c